data/hai/hai reveal 1 intro.txt

```diff
@@ -0,0 +1,953 @@
+			`	The spaceport has massive hangars and advanced repair facilities designed to accommodate the largest Hai warships. Where a fleet has just returned from a recent battle, the hangars are lit with flashes from robotic welders as damaged hull plates are swapped out for fresh ones and weakened joints are reinforced. The actual process of building these warships, however, takes place on the Hai homeworld, where the facilities will not be vulnerable to attack.`
+			`	It does not take long to realize that this conflict really is constant, even if the friendlier Hai on this side seem to be maintaining their defense without apparently taxing their civilization to any great extent.`
+
+# Both tours merge here
+mission "Hai Tour [5] Tour Ends"
+	landing
+	name "Head directly to Hai-home"
+	description "Go to Hai-home and meet whoever wants to speak with you."
+	source "Cloudfire"
+	destination "Hai-home"
+	clearance
+	to offer
+		has "00 Hai Tour [4] Cloudfire: done"
+	on offer
+		conversation
+			branch "skipped tours"
+				has "Hai Witness Unfettered [1]"
+			`After looking around Cloudfire for some time, you stop in the spaceport for a moment to absorb everything you've seen. A screen is playing through news on the wall nearby, and your ears suddenly perk up at the word "marauder".`
+			`	"Yesterday, a marauder terrorized the inhabitants of Allhome. They arrived by posing as a trader to investigate rumors currently circulating in human space about our existence. Hai security failed to identify the danger, and the individual snuck his ship in and landed at the spaceport without any opposition. Then his crew kidnapped nearly four dozen Hai and hijacked a Geocoris before making off with their spoils."`
+				goto "that's awful"
+			label "skipped tours"
+			`Soon after you land on <planet>, you notice a news broadcast on a nearby wall. "Yesterday, a marauder terrorized the inhabitants of Allhome. They arrived by posing as a trader to investigate rumors currently circulating in human space about our existence. Hai security failed to identify the danger, and the individual snuck his ship in and landed at the spaceport without any opposition. Then his crew kidnapped nearly four dozen Hai and hijacked a Geocoris before making off with their spoils."`
+				goto "that's awful"
+			label "that's awful"
+			`	At a table nearby you hear a Hai staff member exclaim, "Oh, that's awful!"`
+			`	The rumors must be spreading further by the day. There's no doubt even more problems than this marauder will follow.`
+			`	This thought has no sooner crossed your mind when your solitude is interrupted by a small group of somewhat official-looking Hai.`
```

Comment:
Ok now that you point this out to me I actually hate this whole sentence. Something feels like it's ass-about-face. Do you reckon you could have a crack at re-doing it entirely?

Reply:
```suggestion
			`	No sooner has this thought crossed your mind than your solitude is interrupted by a small group of somewhat official-looking Hai.`
```

This would be the easiest general fix, just transplanting that phrase to the front.